citizen_report/report/views.py

```diff
@@ -25,26 +25,15 @@ def create(request):
             report.save()
             return redirect('reports')
         else:
-            error = 'something is wrong'
-            return render(request, 'create.html', {'form': ReportForm(), 'error': error})
+            error = 'wrong data in form'
+            return render(request, 'create.html', {'form': ReportForm(request.POST), 'error': error})
     else:
-        error = 'something went wrong'
-        return render(request, 'create.html', {'form': ReportForm(), 'error': error})
+        error = 'method not allowed'
+        # TODO: HttpResponse('405 - do zmiany!!!') or Response(status=status.HTTP_405) EXPLORE DOCS
+        return render(request, 'create.html', {'error': error})
```

Comment:
https://stackoverflow.com/questions/65121653/how-to-raise-or-return-405-method-not-allowed-in-django